modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -2788,10 +2788,10 @@ void ONNXImporter::parseSoftMax(LayerParams& layerParams, const opencv_onnx::Nod
 {
     const std::string& layer_type = node_proto.op_type();
     int axis;
-    if (layerParams.has("opset") && layerParams.get<int>("opset") > 11) {
```

Comment:
Should we update `parseQSoftmax` too? (may be in a separate PR?)

Reply:
Let me try to change it in QSoftmax

Reply:
Please create dedicated PR for parseQSoftmax.